core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -735,6 +735,15 @@ case class ConnectInit(sourceInfo: SourceInfo, loc: Node, exp: Arg) extends Comm
 case class Stop(sourceInfo: SourceInfo, clock: Arg, ret: Int) extends Command
 case class Port(id: Data, dir: SpecifiedDirection)
 case class Printf(sourceInfo: SourceInfo, clock: Arg, pable: Printable) extends Command
+// formal
```

Comment:
Can you please remove the `// formal` and `// end formal` comment? No other IR nodes have that and it seems to provide no additional information.